tests/serverpod_cli_e2e_test/test/generate_watch_test.dart

```diff
@@ -371,6 +371,188 @@ class TestEndpoint extends Endpoint {
     });
   });
 
+  group(
+      'Given an endpoint file in the "lib/src" folder that is changed when generate watch is active',
+      () {
+    var (projectName, commandRoot) = createRandomProjectName(tempPath);
+    var (serverDir, _, _) = createProjectFolderPaths(projectName);
+
+    late Process createProcess;
+    Process? generateWatch;
+    KeywordSearchInStream generateStreamSearch = KeywordSearchInStream(
+      keywords: generateWatchCompletionKeywords,
+    );
+    setUp(() async {
+      // Create project
+      createProcess = await Process.start(
+        'serverpod',
+        ['create', projectName, '-v', '--no-analytics'],
+        workingDirectory: tempPath,
+        environment: {
+          'SERVERPOD_HOME': rootPath,
+        },
+      );
+
+      createProcess.stdout.transform(const Utf8Decoder()).listen(print);
+      createProcess.stderr.transform(const Utf8Decoder()).listen(print);
+
+      var createProjectExitCode = await createProcess.exitCode;
+      assert(
+        createProjectExitCode == 0,
+        'Failed to create the serverpod project.',
+      );
+    });
+
+    tearDown(() async {
+      createProcess.kill();
+      generateWatch?.kill();
+      generateStreamSearch.cancel();
+
+      await Process.run(
+        'docker',
+        ['compose', 'down', '-v'],
+        workingDirectory: commandRoot,
+      );
+
+      while (!await isNetworkPortAvailable(8090)) {}
+    });
+
+    test('then endpoint dispatcher is generated and updated as expected.',
+        () async {
+      // Start generate watch
```

Comment:
Nit: remove comment

Reply:
To add some additional context, I think these comments are redundant especially when it is obvious from the code that is going on. It just adds noise. I think most of these comments can be removed in these files.